packages/steiger/src/models/config/validate-config.spec.ts

```diff
@@ -8,11 +8,12 @@ import { isPlugin } from './raw-config'
 // but with the check method expected to be just any function.
 // The reason why it's needed is that the original check method's .toString method returns [Function: check],
 // but after the validation it changes to [Function: anonymous] and fails the test.
-function expectCheckToBeAnyFunction(config: Config<Array<Rule>>) {
+function expectFunctionsToBeAnyFunction(config: Config<Array<Rule>>) {
```

Comment:
issue: now this name is confusing enough to me that even the comment before it doesn't quite cut it :D

suggestion: how about, instead of checking for equality, we simply make it check that the call to `.parse` doesn't throw? That's what we're trying to test, after all, and I can't come up with a way to neatly do equality checking either

Reply:
Lol, I don't even know why I haven't done that in the first place and ended up inventing a square wheel 🤷‍♂️